src/libraries/System.Threading.RateLimiting/tests/FixedWindowRateLimiterTests.cs

```diff
@@ -71,6 +71,15 @@ public override void InvalidOptionsThrows()
                     Window = TimeSpan.FromMinutes(-2),
                     AutoReplenishment = false,
                 }));
+            Assert.Throws<ArgumentException>(
```

Comment:
Ideally all of these Assert.Throws would instead be AssertExtensions.Throws and have a first argument that's the name of the argument we expect is being flagged as invalid.

Reply:
Wasn't aware of those extensions 👍